Assets/BossRoom/Prefabs/Dungeon/Dungeon Pieces/env_floor7.prefab

```diff
@@ -0,0 +1,112 @@
+%YAML 1.1
```

Comment:
So I'm noticing a considerable amount of smaller floor pieces on top of the newer big floor pieces.

These are just accent pieces, right? I'm curious how it'd look like without them and just rely on the bigger pieces, to double down on the optimization we're gaining.

Reply:
They are just accent pieces, yeah! I think just having the same texture repeated everywhere won't look very good, but let me try modifying the UVs of the big pieces so they have more texture variation!